diff --git a/node_modules/@react-native-community/cli/.DS_Store b/node_modules/@react-native-community/cli/.DS_Store
new file mode 100644
index 0000000..94f64b4
Binary files /dev/null and b/node_modules/@react-native-community/cli/.DS_Store differ
diff --git a/node_modules/@react-native-community/cli/build/tools/config/findDependencies.js b/node_modules/@react-native-community/cli/build/tools/config/findDependencies.js
index 5a58b60..18c1987 100644
--- a/node_modules/@react-native-community/cli/build/tools/config/findDependencies.js
+++ b/node_modules/@react-native-community/cli/build/tools/config/findDependencies.js
@@ -39,7 +39,16 @@ function findDependencies(root) {
     return [];
   }
 
-  const deps = [...Object.keys(pjson.dependencies || {}), ...Object.keys(pjson.devDependencies || {})];
+  const dependencies = Object.keys(pjson.dependencies || []);
+  const subDeps = dependencies.reduce(function(accumulator, currentValue) {
+    if (!currentValue.includes('@sleeperhq')) {
+      return accumulator;
+    }
+    const subRoot = `${root}/node_modules/${currentValue}`;
+    return accumulator.concat(findDependencies(subRoot));
+  }, []);
+
+  const deps = [...Object.keys(pjson.dependencies || {}), ...Object.keys(pjson.devDependencies || {}), ...subDeps];
   return deps;
 }
 
